http/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "testing-code",
+  "version": "1.0.0",
+  "description": "",
+  "main": "",
+  "scripts": {
```

Comment:
add a script to start and build the project

Reply:
done